pyserver.py

```diff
@@ -22,8 +25,8 @@ def do_GET(self):
         self.render(text)
 
     def render(self, text):
-
-        with open('C:\\Users\\Adeon\\PycharmProjects\\VALERA\\template\\index.html', 'r', encoding="utf8") as file:
+        indexPath = os.path.join('.', 'template', 'index.html')
```

Comment:
I'm not sure it will work, but let's try!

Reply:
Ok, let's go